### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by Dandush03**
# Created Library Project
### In this project I used:
- HTML5
- CSS3
- JS
- ES6
- VSCode

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  const inputs = document.getElementById('bookSubmit');
+  const book = new Book();
+  Object.keys(book).forEach(key => {
+    if (key !== 'isReaded' && key !== 'id') {
+      book[key] = inputs[key].value;
+    } else if (key === 'id') {
+      book[key] = myLibrary.length + 1;
+    }
+  });
+  myLibrary.push(book);
```
Comment (shloch) [inline script.js]: Apply minimum security of checking whether user has filled form before submission........
Else we get something like the screenshot below

Screen Shot 2020-04-11 at 5 28 04 PM

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @Dandush03 ,

Great job.........looks perfect and professional

### one more thing to do below before a last thumbs up....happy coding

**Comment by Dandush03** [comment]
#  Thanks for your Code Review :exclamation: :exclamation: 
#### I implement a validation rule for the form

## Happy Coding :computer: 
## Best Regards :exclamation: :exclamation: 
### Daniel Laloush

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (shloch) [inline .eslintrc.json]: You shouldn't have touched this config though, please change and use the config file provided by microverse and re-run for linting errors  
[ [see config here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]

Comment (shloch) [reply .eslintrc.json]: Screen Shot 2020-04-12 at 1 19 39 PM

**Review by shloch — CHANGES_REQUESTED** [review]
GREAT JOB
Screen Shot 2020-04-12 at 1 13 00 PM
Screen Shot 2020-04-12 at 1 13 26 PM

Screen Shot 2020-04-12 at 1 13 38 PM

Screen Shot 2020-04-12 at 1 13 51 PM


## put the linting config back for final thumbs up  😃😃😃😃😃😃

**Comment by Dandush03** [comment]
# Hey :wave: :exclamation: 
## The only difference was that I disabled less things... :exclamation:  I hope now it will suit your needs.

#### Thanks for your time

**Review by shloch — APPROVED** [review]
### we're good to go bro.......on to the next project
## happy coding 👏 👏 👏 👏 👏 

accomplished

**Pull request closed without merging**